src/token_request_handler.ts

```diff
@@ -46,6 +51,24 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
     return (response as TokenErrorJson).error === undefined;
   }
 
+  performRevokeTokenRequest(
+      configuration: AuthorizationServiceConfiguration,
+      request: RevokeTokenRequest): Promise<boolean> {
+    let revokeTokenResponse = this.requestor.xhr<any>({
+      url: configuration.revocationEndpoint + '?' + this.utils.stringify(request.toStringMap()),
+      dataType: 'json',  // adding implicit dataType
+      method: 'GET'
```

Comment:
good point, will fix that as well.
And we need check the response code only: (instead and empty object {}):
> The content of the response body is ignored by the client as all
>   necessary information is conveyed in the response code.

probably I need to change xhr as well to return the response code, and I will extend the revoke class to include param token_type_hint besides token.